mkchimeenv.py

```diff
@@ -106,6 +106,42 @@ def match_opcode(opcode: int) -> Tuple[int, str, bool]:
         return (0, "Unknown", done)
 
 
+def find_requirements(path: str) -> List[Requirement]:
```

Comment:
Question - does this seem too simple?

Reply:
I think it's fine.  It's a fairly simple thing we're trying to do now.  The hard part was all the build-system conversion.